src/main/resources/fileFrom.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
[Place the input and output files into the src/main/resources folder.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder). Fix it for all files